.github/workflows/build_and_deploy.yml

```diff
@@ -0,0 +1,31 @@
+name: Build and Deploy
+on:
+  push:
+    tags:
+    - 'v[0-9]+.[0-9]+.[0-9]+'
+jobs:
+  build_and_deploy:
+    if: github.repository_owner == 'ManageIQ'
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v4
+    - name: Set up Ruby
+      uses: ruby/setup-ruby@v1
+      with:
+        ruby-version: "2.7"
+        bundler-cache: true
+    - name: Docker login
+      run: echo ${{ secrets.DOCKER_REGISTRY_PASSWORD }} | docker login docker.io --password-stdin --username ${{ secrets.DOCKER_REGISTRY_USERNAME }}
+    - name: Build
+      run: bundle exec rake production:release:build[${{ github.ref_name }}]
+    - name: Set up kubectl
+      uses: azure/setup-kubectl@v4
+    - name: Set up ibmcloud CLI
+      uses: IBM/actions-ibmcloud-cli@4fb9b6c
```

Comment:
TODO: This is in progress - I'm working to revamp this action - Eventually when I cut a release I'll switch this to `@v1`

Reply:
@bdunne I released v1, so I've updated this accordingly